specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/examples/ManagedHsm_CreateOrUpdate.json

```diff
@@ -0,0 +1,89 @@
+{
+  "parameters": {
+    "resourceGroupName": "hsm-group",
+    "name": "hsm1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2022-11-01",
+    "parameters": {
+      "properties": {
+        "tenantId": "00000000-0000-0000-0000-000000000000",
+        "initialAdminObjectIds": [
+          "00000000-0000-0000-0000-000000000000"
+        ],
+        "enableSoftDelete": true,
+        "softDeleteRetentionInDays": 90,
+        "enablePurgeProtection": true
```

Comment:
Please set all the `enablePurgeProtection` flags to `false` in examples. Se context here: https://github.com/Azure/azure-rest-api-specs/pull/21182

Reply:
Done.